src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    String getDraw();
```

Comment:
```suggestion
    void draw();
```

давай сделаем этот метод void так какк он должен вывести информацию о фигуре на экраи и ничего не возвращать